src/internal/joiExtended.ts

```diff
@@ -126,6 +127,38 @@ export const joi = Joi
     };
   })
 
+  .extend((joi: Joi.Root) => {
+    return {
+      type: 'token',
+      base: joi.any(),
+      messages: {
+        'token.transform': 'Transformation failed for {{#label}}'
+      },
+      rules: {
+        transform: {
+          method: function(): any {
+            // @ts-ignore
+            return this.$_addRule('transform');
+          },
+          validate: function(value: any, helpers: any) {
+            const chainIdd = joi.ref('chainId')
+          const chainId = 1;
+
+            if (!chainId) {
+              return helpers.error('token.transform', { label: helpers.state.path });
+            }
+            try {
+              return toTokenWithDecimalsArgs(value, chainId);
+            } catch (e) {
+            throw e
+              // return helpers.error('token.transform', { label: helpers.state.path });
```

Comment:
for now maybe remove this whole unused try/catch

Reply:
yeah sorry was for debugging. removing.